airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3Destination.java

```diff
@@ -134,12 +134,23 @@ public AirbyteMessageConsumer getConsumer(final JsonNode config,
         config.has("uploading_method") ? EncryptionConfig.fromJson(config.get("uploading_method").get(JdbcUtils.ENCRYPTION_KEY)) : new NoEncryption();
     final JsonNode s3Options = findS3Options(config);
     final S3DestinationConfig s3Config = getS3DestinationConfig(s3Options);
+
+    final int numFileBuffers = s3Options.has("file_buffer_count") ? s3Options.get("file_buffer_count").intValue() : 100;
```

Comment:
This line will force existing jobs to use `100` concurrent buffers instead of `10` (current behavior). To keep the changes safe for those who are already using this connector, please change the fallback value to `10`.

Reply:
Addressed in 9d1043b